### PR TITLE
[backport/v1.21] release: 1.21.4 (+ 1.21.3 patch stack)

### PR DESCRIPTION
**Pull request by phlax**
Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

**Comment by phlax** [comment]
im guessing we want to add the next releease _with_ this PR (im happy to do after if preferred)

cc @pradeepcrao 

i need to add the coverage fix also by the looks of it

**Comment by mattklein123** [comment]
Yeah I would suggest that we actually cut a release again, otherwise we have a stale branch. The option would be to immediately set it up for the next release in this PR?

/wait

**Comment by phlax** [comment]
> he option would be to immediately set it up for the next release in this PR?

yep, ill add

File: docs/root/version_history/current.rst [hunk 1]
```diff
@@ -1,5 +1,5 @@
-1.21.3 (June 9, 2022)
-=====================
+1.21.4 (June 14, 2022)
```
Comment (mattklein123) [inline docs/root/version_history/current.rst:1]: nit: should we have some release note to say why we are cutting this?

/wait

Comment (phlax) [reply docs/root/version_history/current.rst:1]: ill include info about the docker image update...

**Review by mattklein123 — APPROVED** [review]
Thanks!

**Comment by phlax** [comment]
@mattklein123 could we release this one today? if not, i should update the release date

**Pull request merged**